pyproject.toml

```diff
@@ -4,7 +4,7 @@ version = "0.4.5"
 description = "Federated blocklist management for Mastodon"
 readme = "README.md"
 license = {file = "LICENSE"}
-requires-python = ">=3.6"
+requires-python = ">=3.8"
```

Comment:
Why do we need to drop support for Python 3.6/3.7?

An issue was raised in the past (https://github.com/eigenmagic/fediblockhole/issues/41) regarding version support for older distributions. Dropping support for Ubuntu bionic (18.04LTS) may well be warranted now that we're up to 24.04LTS so 18.04 is more than n-2 LTS revisions behind. I just want to do so purposefully so Fediblockhole doesn't cause headaches for admins unnecessarily.

Reply:
Because versions of Python lower than 3.10 are no longer receiving even security patches, must less bugfixes. Supporting older versions of Python acts as a drag on wider adoption of the newer versions, which is less of a problem for this package per se and more of a "support the larger developer community" question. I'm not actually sure we can use 3.6 with uv (see the comment there, too)

Reply:
Fair enough. That makes it easier to drop Python 3.6 since it would appear no one is using it. Another argument in support of merging in the PR that adds CI!